evalml/pipelines/pipeline_base.py

```diff
@@ -197,16 +193,13 @@ def compute_estimator_features(self, X, y=None):
     def _compute_features_during_fit(self, X, y):
         X_t = X
         for component in self.component_graph[:-1]:
-            self.input_feature_names.update({component.name: list(pd.DataFrame(X_t))})
+            self.input_feature_names.update({component.name: list(X_t.columns)})
```

Comment:
So we're relying on the other pipeline code to have done this conversion to pandas DF for us before `PipelineBase.fit` is called?

Reply:
Yeah... confusing where things link but, `fit` (where conversion code is) --> `_fit` --> `_compute_features_during_fit`